csp-guru/datapackage.json

```diff
@@ -1,11 +1,161 @@
+                        "description": " how many hours the turbine can run on stored heat alone"
+                    },
+                    {
+                        "name": "Field_size",
+                        "description": "the mirror area in square meters"
+                    },
+                    {
+                        "name": "Cooling",
+                        "description": "Cooling technology: dry, wet ,or hybrid"
+                    },
+                    {
+                        "name": "Expected_generation",
+                        "description": "Yearly power output [GWh/a]"
+                    },
+                    {
+                        "name": "Load_factor",
+                        "description": "percentage of maximum yearly power output, increases with storage"
+                    },
+                    {
+                        "name": "Total_cost",
+                        "description": "in thousand local currency"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015USD",
+                        "description": "investment cost per MW [2015 USD], discounted"
+                    },
+                    {
+                        "name": "Specific_investment_cost_2015EUR",
+                        "description": "investment cost per MW [2015 EURO], discounted"
+                    },
+                    {
+                        "name": "Developer",
+                        "description": "Company that developed the project"
+                    },
+                    {
+                        "name": "HCE_manufacturer",
+                        "description": "Heat Collecting Element manufacturer"
+                    },
+                    {
+                        "name": "SCA_manufacturer",
+                        "description": "Solar Collector Assemblies manufacturer"
+                    },
+                    {
+                        "name": "Financed_through",
+                        "description": "How the owner makes money from the generated electricity: PPA, FiT, Spot Sales"
+                    },
+                    {
+                        "name": "Remuneration",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], local currency"
+                    },
+                    {
+                        "name": "Remuneration_2015USD",
+                        "description": "indicates the level of FiT or PPA [ct/kWh], 2015 USD "
+                    },
+                    {
+                        "name": "Remuneration_2015EUR",
+                        "description": "indicates the level of FiT or PPA in [ct/kWh], 2015 EURO "
+                    },
+                    {
+                        "name": "LCOE_5_2015USD",
+                        "description": "Levelized Cost of Electricity with 5 years payback [ct/kWh], 2015 USD discounted"
```

Comment:
This is not sufficient to understand where the number comes from. But that should not be part of this change I guess.